api/src/index.ts

```diff
@@ -46,43 +47,60 @@ async function getRootIdentityId(config: Config): Promise<string> {
 	return null;
 }
 
+process.on("uncaughtException", function(err) {
+    // clean up allocated resources
+    // log necessary error details to log files
+	console.log("ERRORE!!!");
```

Comment:
I think the logger should be used instead of console.log. in addition there is a typo

Reply:
done